geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/export/ExportCommandTools.scala

```diff
@@ -0,0 +1,78 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.tools.export
+
+import java.io._
+import java.util.zip.GZIPOutputStream
+
+import com.beust.jcommander.ParameterException
+import com.typesafe.scalalogging.LazyLogging
+import org.geotools.data.simple.SimpleFeatureCollection
+import org.geotools.data.{DataStore, Query}
+import org.geotools.filter.text.ecql.ECQL
+import org.opengis.filter.Filter
+
+import scala.collection.JavaConverters._
+import scala.util.{Failure, Success, Try}
+
+trait ExportCommandTools extends LazyLogging {
```

Comment:
might be good to make this an object instead of a trait, since these all seem to be static methods. If they're just internal helper methods, I think that's a cleaner way to get the functionality.